docs/source/terms_and_model.rst

```diff
@@ -1411,37 +1286,84 @@ Absolute copy number counts may not be smaller than zero.
      - Type
      - Limits
      - Description
+   * - value
+     - number
+     - 1..1
+     - A simple number value
    * - type
      - string
      - 1..1
-     - MUST be "AbsoluteCopyCount"
+     - MUST be "Number"
+
+.. _DefiniteRange:
+
+DefiniteRange
+#############
+
+**Computational Definition**
+A bounded, inclusive range of numbers.
+
+**Information Model**
+
+.. list-table::
+   :class: reece-wrap
+   :header-rows: 1
+   :align: left
+   :widths: auto
+
+   * - Field
+     - Type
+     - Limits
+     - Description
    * - min
-     - integer
+     - number
      - 1..1
-     - minimum value; inclusive
+     - A simple number value representing the range minimum
    * - max
-     - integer
+     - number
      - 1..1
-     - maximum value; inclusive
-
-**Implementation Guidance**
+     - A simple number value representing the range maximum
+   * - type
+     - string
+     - 1..1
+     - MUST be "DefiniteRange"
 
-* If both ``min`` and ``max`` MUST satisfy ``min
```

Comment:
do we need these impl guidance points? Is this a constraint in our schema or is this something we recommend? We should clarify and be consistent as to what level of Impl Guidance we apply across all our types.
```
** Implementation Guidance **
* If the `min` and `max` values are equal then the entire concept will be normalized to a `Number` type with a single `value` equal to the `min` and `max` value. This will assure there is one normalized representation of a single point that may be unintentionally expressed as a DefiniteRange.

Reply:
Let me know if you think my Impl Guidance rule above is accurate. It seems to make sense to me that we want our "normalizing routines" to reduce alternative representations down to a common form.

I suppose we could simply fail validation if max <= min. But we may want to allow folks to represent min and max as equal even though they are equivalent to Number. I could go either way I suppose.

Reply:
Since this comment is unrelated to aligning terminology and schema, please reraise elsewhere.